### PR TITLE
[Workers] Adds New Python Examples

### PR DESCRIPTION
**Pull request by megaconfidence**
Moooarr Python examples!

### CODE REVIEW

File: content/workers/examples/bulk-redirects.md [hunk 1]
```diff
@@ -66,5 +66,31 @@ export default {
...
+
+    redirect_map = {
+      "/bulk1": "https://" + external_hostname + "/redirect2",
+      "/bulk2": "https://" + external_hostname + "/redirect3",
+      "/bulk3": "https://" + external_hostname + "/redirect4",
+      "/bulk4": "https://google.com",
+      }
+
+    url = URL.new(request.url)
+    location = redirect_map[url.pathname]
```
Comment (hoodmane) [inline content/workers/examples/bulk-redirects.md]: You'll get an `IndexError` if it's not in the dictionary:
```suggestion
    location = redirect_map.get(url.pathname, None)
```

**Review by kodster28 — APPROVED** [review]
Looks good from a content side + general perusing the code, requested a tech review internally.

**Review by garrettgu10 — APPROVED** [review]
LGTM assuming you've successfully tried running and deploying them. Thanks!

File: content/workers/examples/block-on-tls.md [hunk 2]
```diff
@@ -65,4 +65,17 @@ export default {
...
 {{</tab>}}
-{{</tabs>}}
+{{<tab label="py">}}
+
+```py
+from js import Response, fetch
+
+async def on_fetch(request):
+    tls_version = request.cf.tlsVersion
+    if tls_version != "TLSv1.2" and tls_version != "TLSv1.3":
```
Comment (hoodmane) [inline content/workers/examples/block-on-tls.md]: Maybe:
```suggestion
    if tls_version not in ("TLSv1.2", "TLSv1.3"):
```

**Review by hoodmane — APPROVED** [review]
Thanks @megaconfidence!

File: content/workers/examples/bulk-origin-proxy.md [hunk 3]
```diff
@@ -69,5 +69,30 @@ export default {
 } satisfies ExportedHandler;
 ```
 
+{{</tab>}}
+{{<tab label="py">}}
+
+```py
+from js import Response, fetch, URL
```
Comment (KianNH) [inline content/workers/examples/bulk-origin-proxy.md]: nit: `Response` is unused

```suggestion
from js import fetch, URL
```

**Review by irvinebroque — APPROVED** [review]

**Comment by megaconfidence** [comment]
Thanks for the feedback everyone! Merging now...

**Pull request merged**